Core/Utilities/Graphs/Legend.py

```diff
@@ -181,8 +178,8 @@ def draw(self):
       else:
         num = None
       color = self.palette.getColor(label)
-      row = nc % nRows
-      column = nc / nRows
+      row = int(nc % nRows)
```

Comment:
Here `int` is not needed

Reply:
it is needed for python3

Reply:
The `%` operator will give always an `int` in output.

Reply:
Federico, I wrote integration tests in python2 and after I executed the tests in python3 and I spent lot of times to fix the bugs and now you tell it is always int. 
python3:
```
>>> 3/2
1.5
>>> 
```
python2
```
>>> 3/2
1
>>> 
```

Reply:
yes but in this line (181) there's no division: there's the modulo operator.

Reply:
sorry. the line above was the problematic one.